src/main/java/org/folio/marc/migrations/services/jdbc/InstanceJdbcService.java

```diff
@@ -0,0 +1,87 @@
+package org.folio.marc.migrations.services.jdbc;
+
+import java.util.List;
+import java.util.UUID;
+import lombok.extern.log4j.Log4j2;
+import org.folio.marc.migrations.domain.entities.MarcRecord;
+import org.folio.spring.FolioExecutionContext;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@Service
+public class InstanceJdbcService extends JdbcService {
+
+  private static final String CREATE_INSTANCE_VIEW_SQL = """
+    CREATE OR REPLACE VIEW %s.instance_view
+          AS
+        SELECT *
+          FROM %s_mod_inventory_storage.instance;
```

Comment:
I think we should filter Instances by source. We can update only instances with source=MARC